src/Proto.Cluster/Gossip/GossipActor.cs

```diff
@@ -4,84 +4,57 @@
 // </copyright>
 // -----------------------------------------------------------------------
 using System;
-using System.Collections.Generic;
 using System.Collections.Immutable;
-using System.Linq;
 using System.Threading.Tasks;
-using Google.Protobuf;
-using Google.Protobuf.WellKnownTypes;
 using Microsoft.Extensions.Logging;
 using Proto.Logging;
-using Proto.Remote;
 
 namespace Proto.Cluster.Gossip
 {
     public class GossipActor : IActor
     {
-        private readonly TimeSpan _gossipRequestTimeout;
         private static readonly ILogger Logger = Log.CreateLogger<GossipActor>();
-        private long _localSequenceNo;
-        private GossipState _state = new();
-        private readonly Random _rnd = new();
-        private ImmutableDictionary<string, long> _committedOffsets = ImmutableDictionary<string, long>.Empty;
-        private ImmutableHashSet<string> _activeMemberIds = ImmutableHashSet<string>.Empty;
-        private Member[] _otherMembers = Array.Empty<Member>();
-        private readonly ConsensusChecks _consensusChecks = new();
+        private readonly TimeSpan _gossipRequestTimeout;
+        private readonly GossipFoo _foo;
```

Comment:
We need a name for this, the actual gossip implementation without actor code.
Gossip Foo is not a good name :D